### PR TITLE
refactor: Isolate func and model

### PR DESCRIPTION
**Pull request by jing-zhi**
feat:add confirmation button for batch deletion

#### What this PR does / why we need it:

refactor: Isolate func and model
feat:add confirmation button for batch deletion

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link]()

#### Specified Reviewers:

/assign @chengjoey @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      refactor: Isolate func and model，feat:add confirmation button for batch deletion        |
| 🇨🇳 中文    |       重构，隔离函数与结构体，批量删除-增加确认弹窗    |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/model.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	"github.com/erda-project/erda/bundle"
+)
+
+//	batchOperationTipModal: {
+//	       type: 'Modal',
+//	       props: {
+//	         status: 'warning',
+//	         content: '您将解冻以下节点：\n cn-hongkong.172.15.3.3 \ncn-hongkong.172.15.3.3',
+//	         title: '提示',
+//	       },
```
Comment (chengjoey) [inline internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/model.go:32]: delete useless comment

File: internal/apps/dop/providers/efficiency_measure/efficiency_query.go [hunk 2]
```diff
@@ -126,6 +126,9 @@ func (p *provider) queryPersonalEfficiency(rw http.ResponseWriter, r *http.Reque
 	}
 
 	rawSql := p.makeEfficiencyBasicSql(req)
+	fmt.Println("————————————————————————————打印sql————————————————————————————")
```
Comment (chengjoey) [inline internal/apps/dop/providers/efficiency_measure/efficiency_query.go:129]: delete useless codes

File: internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/render.go [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+	req.ProjectID = projectID
+	k, err := bot.issueSvc.BatchDeleteIssues(bot.ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	return k, err
+}
+
+// Transfer transfer a to b with json, kind of b must be pointer
+func Transfer(a, b interface{}) error {
```
Comment (chengjoey) [inline internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/render.go:141]: `Transfer` is necessary to add unit-test

Comment (jing-zhi) [reply internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/render.go:141]: thank you for pointing out

File: internal/apps/dop/providers/efficiency_measure/efficiency_query.go (hunk 2 above)
Comment (jing-zhi) [reply internal/apps/dop/providers/efficiency_measure/efficiency_query.go]: thank you for pointing out

File: internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/model.go (hunk 1 above)
Comment (jing-zhi) [reply internal/apps/dop/component-protocol/components/issue-manage/batchOperationTipModal/model.go]: thank you for pointing out

**Review by chengjoey — APPROVED** [review]

**Pull request merged**